opendevin/controller/agent_controller.py

```diff
@@ -86,7 +86,7 @@ def __init__(
         )
 
         # state from the previous session, state from a parent agent, or a fresh state
-        self._set_initial_state(
```

Comment:
Minor thing, can we please continue to name it `set_initial_state`? It's just that `set_state` tells me that it's a setter for state, and that's not what it is. It should never be called during the application normal run, only at the initialization stages. (init/restore/resets)

Reply:
okay